### PR TITLE
UI upgrades

### PR DESCRIPTION
**Pull request by amiraliemami**
Reviewer: @sidravi1 
Round 2: @suzinyou 
Estimate: 20mins

---

## Ticket

Fixes: [781](https://idinsight.atlassian.net/browse/AAQ-781), [798](https://idinsight.atlassian.net/browse/AAQ-798)

### Goal

- Cleanup UI and make it more responsive to different screen sizes
- Add page titles and descriptions to content and UD pages.

### Changes

- See commits

### Future Tasks

- Change dashboard tables to use MUI Table components and make all dashboard components into a grid system to they can better serve smaller screen sizes.

## How has this been tested?

- Dev
- Docker compose

## Checklist

Fill with `x` for completed. 

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts

### CODE REVIEW

File: admin_app/src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -52,50 +52,56 @@ const Dashboard: React.FC = () => {
...
-            selectedDashboardPage={dashboardPage}
-          />
-        </ClickAwayListener>
+    <Box
+      sx={{
+        display: "flex",
+        marginTop: 4,
+        flexDirection: "row",
+        minWidth: "900px",
+        maxWidth: "1500px",
```
Comment (sidravi1) [inline admin_app/src/app/dashboard/page.tsx:61]: This line adds a gray space on my laptop. Can we remove the `maxWidth` or make it 1900px? That seems to render well on my laptop

image

File: admin_app/src/app/content/components/DownloadModal.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ import Papa from "papaparse";
 import { useState } from "react";
 import { LoadingButton } from "@mui/lab";
 import { Content } from "../edit/page";
+import { Layout } from "@/components/Layout";
 
 const MAX_CARDS_TO_FETCH = 200;
```
Comment (sidravi1) [inline admin_app/src/app/content/components/DownloadModal.tsx:19]: Apparently `ContentDownload` is declared on line 20 but never used. Can we delete it?

File: admin_app/src/app/content/page.tsx [hunk 3]
```diff
@@ -97,48 +112,37 @@ const CardsPage = () => {
...
-                    setFilterTags(updatedTags);
-                  }}
-                  renderInput={(params) => (
-                    <TextField {...params} variant="standard" label="Filter by tags" />
-                  )}
-                  sx={{ width: "80%", color: appColors.white }}
-                />
-              </Layout.FlexBox>
-            </Layout.FlexBox>
             <CardsUtilityStrip
```
Comment (sidravi1) [inline admin_app/src/app/content/page.tsx:139]: I don't know how I feel about the left aligned design. But happy to go with it if everyone thinks it' better.

**Review by sidravi1 — APPROVED** [review]
Thanks @amiraliemami. Look good. Left a couple of small comments

File: admin_app/src/app/content/page.tsx (hunk 3 above)
Comment (amiraliemami) [reply admin_app/src/app/content/page.tsx:139]: Yeah decided this with Anh and Tanmay on a design call so will leave it for now if that's okay. It's the cleanest/most comprehensible grouping that allows for a page title and description I think.

File: admin_app/src/app/dashboard/page.tsx (hunk 1 above)
Comment (amiraliemami) [reply admin_app/src/app/dashboard/page.tsx:61]: Increased to 1900. Adding grey space was my intended behaviour. Should the dashboard page itself be centered do you think? Sidebar must stick to the left but contents of the dashboard could be centered... @Tanmay-97 @suzinyou

File: admin_app/src/app/content/components/DownloadModal.tsx (hunk 2 above)
Comment (amiraliemami) [reply admin_app/src/app/content/components/DownloadModal.tsx:19]: It should have been used as the type for `list_json_content_download` - added now

**Review by sidravi1 — APPROVED** [review]
Thanks @amiraliemami

**Pull request merged**